tools/osx-uninstall.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+# Added to help for removal of node on OSX
```

Comment:
I do not think this will work for brew as brew installs in their own sub directories. 

Updated the comment and file name as requested.